packages/binding.if/spec/ifBehaviors.ts

```diff
@@ -27,7 +27,8 @@ import {bindings as coreBindings} from '@tko/binding.core'
 import '@tko/utils/helpers/jasmine-13-helper'
 
 describe('Binding: If', function () {
-  beforeEach(jasmine.prepareTestNode)
+  var testNode : HTMLElement
```

Comment:
prefer `let` to avoid hoisting?

Reply:
I would fix this globally with eslint in another PR afterwards. It's already prepared in @auge19's branch.

Reply:
see https://github.com/Auge19/tko/pull/6